tgui/packages/tgui/interfaces/Quickload_SOM/Types.tsx

```diff
@@ -0,0 +1,36 @@
+export type Loadout =
+  {
```

Comment:
why is the curly brace on the next line we usually put it on the same line as the = (you even do it for the last one in this file)
applies to the rest here too

Reply:
Just saw the second file applies doubly then

Reply:
done.